test/ExecutionFramework.h

```diff
@@ -68,6 +68,15 @@ class ExecutionFramework
 		std::map<std::string, Address> const& _libraryAddresses = std::map<std::string, Address>()
 	) = 0;
 
+	virtual bytes const& multiSourceCompileAndRunWithoutCheck(
+		std::map<std::string, std::string> const& _sourceCodeMap,
```

Comment:
```suggestion
		std::map<std::string, std::string> const& _sources,
```
(the type of the variable should not be encoded in the name)